discovery-infra/utils.py

```diff
@@ -102,6 +102,29 @@ def get_libvirt_nodes_mac_role_ip_and_name(network_name):
         raise
 
 
+def wait_for_cvo_available():
+    waiting.wait(
+        lambda: is_cvo_available(),
+        timeout_seconds=3600,
+        sleep_seconds=20,
+        waiting_for="CVO to become available",
+    )
+
+
+def is_cvo_available():
+    try:
+        res = subprocess.check_output("kubectl --kubeconfig=build/kubeconfig get clusterversion -o json", shell=True)
+        conditions = json.loads(res)['items'][0]['status']['conditions']
+        for condition in conditions:
```

Comment:
maybe we should print the output here? Otherwise it will be hard to debug, no?

Reply:
the main debugging will be done by the must-gather.  right now, even when we have issues with etcd, what we will see in the conditions is: multiple errors with api server endpoint oath. It does not tell much

Reply:
u can look on failure status, u will all those errors

Reply:
i think we miss logs here